package.json

```diff
@@ -43,7 +43,7 @@
     "chart.js": "^2.3.0",
     "clear-cut": "^2.0.2",
     "coffee-script": "1.12.7",
-    "color": "^0.7.3",
+    "color": "^3.1.2",
```

Comment:
You should revert this

Reply:
@dhairya2019 Update on documentation should not contain dependancy changes. The change to `package.json` should be reverted.